filament/src/materials/defaultMaterial0.mat

```diff
@@ -1,7 +1,8 @@
 material {
     name : "Filament Default Material",
     shadingModel : unlit,
-    featureLevel: 0
+    featureLevel: 0,
+    variantFilter : [ stereo ]
```

Comment:
I'm starting to wonder if we shouldn't have a `variant` parameter that **adds** a variant. we could make `stereo` off by default.  otherwise I feel like 99% of the apps will need this filter.

Reply:
Let me experiment with that and see how feasible it would be.

Reply:
I think we can add a new `variantInclude` parameter.

Also, I realized I need to update MaterialBuilder to filter out the stereo variant automatically for feature level 0. So this change isn't strictly necessary.